Dockerfile

```diff
@@ -535,6 +535,11 @@ ADD patches/sitecustomize.py /root/.local/lib/python3.6/site-packages/sitecustom
 # Set backend for matplotlib
 ENV MPLBACKEND "agg"
 
+# Set up pip to enable pip install.
+ADD patches/kaggle_bashrc /root/.bashrc
+# Patch the system-wide bashrc file for non-root users.
+RUN cat /root/.bashrc >> /etc/bash.bashrc
```

Comment:
Do root shells not load `/etc/bash.bashrc`? If they do load it, why not just patch the system-wide bashrc?

Reply:
Done.